modules/ocf_ldap/files/ocf.schema

```diff
@@ -54,9 +54,23 @@ attributetype ( 1.3.6.1.4.1.41759.2.2.1
         SYNTAX 1.3.6.1.4.1.1466.115.121.1.26
         SINGLE-VALUE )
 
+attributetype ( 1.3.6.1.4.1.41759.2.2.2
+        NAME 'cname'
+        DESC 'DNS CNAME record'
+        EQUALITY caseIgnoreMatch
+        SUBSTR caseIgnoreSubstringsMatch
+        SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+
+attributetype ( 1.3.6.1.4.1.41759.2.2.3
+        NAME 'supportsIPv6'
+        DESC 'Whether the host supports IPv6'
+        EQUALITY booleanMatch
+        SYNTAX 1.3.6.1.4.1.1466.115.121.1.7 )
+
 objectclass ( 1.3.6.1.4.1.41759.2.1.1
         NAME 'ocfDevice'
         DESC 'Attributes for OCF hosts'
         SUP ( ipHost $ ieee802Device $ puppetClient )
         AUXILIARY
-        MUST ( cn $ ipHostNumber $ environment $ type ) )
+        MUST ( cn $ ipHostNumber $ type )
+        MAY ( environment $ cname $ supportsIPv6 ) )
```

Comment:
eventually supportsIPv6 would be made mandatory (`MUST`) but we need to do this during the transition period.